### PR TITLE
Feat/whatsnew

### PR DESCRIPTION
**Pull request by TanmayJain92**
## Purpose

- Add What's New Section
- Update Gateway reference to Tracker
- Update User flow issues

### CODE REVIEW

File: src/layout/TopBar/TopBarStyles.css [hunk 1]
```diff
@@ -123,6 +123,89 @@
...
+
+.whatsNewSubTitle {
+  font-size: 16px;
+  color: var(--color-palette-background-black);
+  text-align: center;
+  margin-top: -10px;
+}
+
+.whatsNewDialogContent {
+  padding: 30px 0px 50px 0px;
```
Comment (patelradhika) [inline src/layout/TopBar/TopBarStyles.css:151]: Change this to be `30px 20px 50px 20px`

**Review by patelradhika — CHANGES_REQUESTED** [review]
image

Go back to sign in link does not take the user back to login page.

File: src/layout/TopBar/TopBar.js [hunk 2]
```diff
@@ -66,6 +74,14 @@ const TopBar = ({
     org_uuid = user.organization.organization_uuid;
   }
 
+  useEffect(() => {
+    if (!_.isEmpty(window.sessionStorage.getItem('isWhatsNewShown'))) {
```
[the remark this answers is not in the document]
Comment (TanmayJain92) [reply src/layout/TopBar/TopBar.js]: Done

File: src/layout/TopBar/TopBarStyles.css (hunk 1 above)
Comment (TanmayJain92) [reply src/layout/TopBar/TopBarStyles.css]: Done

**Review by patelradhika — APPROVED** [review]
Looks good

**Pull request merged**